<launch>
    <param name="joint_list" type="str" value="shoulder_pan_joint shoulder_lift_joint elbow_joint wrist_1_joint wrist_2_joint wrist_3_joint"/>
    <param name="k_to_controller_topic" type="str" value="/ee_values"/>
    <param name="controller_to_k_topic" type="str" value="/joint_values"/>
    <param name="model_path" type="str" value="/daniel/Desktop/ur5-rl/DQ_6.0/rl_model_54000_steps"/>


    <node name="k_module" pkg="ur5_rl" type="k_node.py" output="screen"/>
    <node name="rl_controller" pkg="ur5_rl" type="rl_controller.py" output="screen"/>
    <node name="object_manager" pkg="ur5_rl" type="manageObjects.py" output="screen"/>

    <include file="$(find ur5_rl)/launch/rl_gazebo.launch"/>
</launch>